scripts/test-backend.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# file: test-backend.sh
+# description: Test maven and image build
```

Comment:
These checks should be a part of their respective commands i.e. 
[ ] create scripts/package-backend.sh
[ ] add to end of scripts/image-backend.sh